SemanticUI-less blocks widget (ObjectListWidget) + Improve and reorder documentation using diataxis. (#486)
kitconcept/volto-light-theme: docs/conceptual-guides/customization-behavior.md

Reviewer: I think this would make more sense as a how-to guide or several different how-to guides (how to customize the header, how to customize the footer, how to customize themes/colors, etc...) The reader is presumably trying to learn how to change these things in their site, not just see a list of what they can change.

Of course, it'll probably be clearer how to document it once we create the kitconcept.sitebuilder package with these behaviors.

Author: @davisagli agree. Maybe let's put this file in the fridge as reference, but remove the link from the index. We can retake it from here when `kitconcept.sitebuilder` is in place.

Author: Removed from TOC for now.

# Customization behavior

VLT supports site customization via the add-on ...

> This add-on has a behavior that can be applied to the Plone Site root or to any object that implement `INavigationRoot` interface (eg. language root folder.)

Reviewer 2: ```suggestion
This add-on has a behavior that can be applied to the Plone site root or to any object that implements the `INavigationRoot` interface, such as the language root folder.
```

> This add-on uses the `navroot` expander to get all the extra fields that these customizations.

Reviewer 2: What? I think your typing didn't keep up with your brain.

Author: Done, how about now?

> Then VLT access this data to display the customizations in the right place.

Reviewer 2: ```suggestion
Then VLT accesses this data to display the customizations in the right place.
```

## Header

### Site logo

> You can define the site logo not only for the main site, but also for the subsites provided by content types that implement `INavigationRoot` interface.

Reviewer 2: ```suggestion
You can define the site logo for the main site as well as the subsites provided by content types that implement the `INavigationRoot` interface.
```

### Complementary logo

You can define a complementary logo in the header, for convenience purposes.
This logo shows up on the right-most side of the header.

### Intranet Flag

> If you use the intranet header (`config.settings.intranetHeader`) the intranet flag is the text in the grey pill at the top of the header.

Reviewer 2: ```suggestion
If you use the intranet header (`config.settings.intranetHeader`), the intranet flag is the text in the grey pill at the top of the header.
```

### Actions

You can define the actions at the top of the header.

> These are links to other pages, defined by (title, target URL and if it opens in a new tab or not).

Reviewer 2: I'm not sure of the intent, with the parentheses thrown in for some unknown reason.

## Theming

### Navigation text color

You can customize the navigation text color in this widget.

> ### Fat menu / Breadcrumbs text color

Reviewer 2: ```suggestion
### Fat menu and breadcrumbs text color
```

> You can customize the fat menu (the menu that unfolds if you click on any item of the site sections) text color in this widget.

Reviewer 2: ```suggestion
The fat menu is the menu that unfolds when you click on any item in the site section navigation.
You can customize the fat menu text color in this widget.
```

> It applies also to the breadcrumbs text color.

Reviewer 2: ```suggestion
It also applies to the breadcrumbs text color.
```

### Fat menu background color

You can customize the fat menu background color in this widget.

### Footer font color

You can customize the footer font color in this widget.

### Footer background color

You can customize the footer background color in this widget.

## Footer

### Footer links

The footer can contain additional links defined in this widget.

> These are links to other pages, defined by (title, target URL and if it opens in a new tab or not).

Reviewer 2: Same as previous comment about parentheses.

Author: Done.

### Footer logos

The footer can contain a list of logos.

> These are images defined in the site with links to other pages, defined by (title, logo image, target URL and if it opens in a new tab or not).

Reviewer 2: Again.

Author: Done

> Their appearance can be further customized by defining the size of the logos (small or large), and the width of the their container (default or layout).

Reviewer 2: ```suggestion
Their appearance can be further customized by defining the size of the logos, as either `small` or `large`, and the width of the their container, either `default` or `layout`.
```

# Customization behavior

> VLT supports site customization via the add-on ...

Reviewer 2: ...?

> VLT supports site customization via the add-on ...

Reviewer 2: ```suggestion
% TODO name the add-on
VLT supports site customization via the add-on ...
```

This add-on has a behavior that can be applied to the Plone site root or to any object that implements the `INavigationRoot` interface, such as the language root folder.
This add-on also enhances the `@navroot` endpoint to provide the customization fields to the frontend.
Then VLT accesses this data to display the customizations in the right place.

## Header

### Site logo

You can define the site logo for the main site as well as the subsites provided by content types that implement the `INavigationRoot` interface.

### Complementary logo

You can define a complementary logo in the header, for convenience purposes.
This logo shows up on the right-most side of the header.

### Intranet Flag

If you use the intranet header (`config.settings.intranetHeader`), the intranet flag is the text in the grey pill at the top of the header.

### Actions

You can define the actions located at the top right of the header.

> These are links to other pages, each of one defined by these fields: a title, a target URL and a boolean in case it should open in a new tab or not.

Reviewer 2: ```suggestion
These are links to other pages, each of which is defined by the fields for a title, a target URL, and a boolean for whether or not to open the link in a new tab.
```

## Theming

### Navigation text color

You can customize the navigation text color in this widget.

### Fat menu and breadcrumbs text color

The fat menu is the menu that unfolds when you click on any item in the site section navigation.
You can customize the fat menu text color in this widget.
It also applies to the breadcrumbs text color.

### Fat menu background color

You can customize the fat menu background color in this widget.

### Footer font color

You can customize the footer font color in this widget.

### Footer background color

You can customize the footer background color in this widget.

## Footer

### Footer links

The footer can contain additional links defined in this widget.

> These are links to other pages, each of one defined by these fields: a title, a target URL and a boolean in case it should open in a new tab or not.

Reviewer 2: ```suggestion
These are links to other pages, each of which is defined by the fields for a title, a target URL, and a boolean for whether or not to open the link in a new tab.
```

### Footer logos

The footer can contain a list of logos.

> These are images defined in the site with links to other pages, each of one defined by these fields: a title, a logo image, a target URL and a boolean in case it should open in a new tab or not.

Reviewer 2: ```suggestion
These are images defined in the site with links to other pages, each of which is defined by the fields for a title, a target URL, and a boolean for whether or not to open the link in a new tab.
```
